### PR TITLE
separate kafka consumer from nakadi consumer

### PR DESCRIPTION
**Pull request by adyach**
the commits removes confusion in the layer between Kafka consumer and Nakadi consumer

### CODE REVIEW

**Comment by adyach** [comment]
👍

File: core-common/src/main/java/org/zalando/nakadi/repository/kafka/NakadiKafkaConsumer.java
Comment (a1exsh) [inline core-common/src/main/java/org/zalando/nakadi/repository/kafka/NakadiKafkaConsumer.java:1]: Looks like we didn't remove timeline map from this class.
It also feels like the Low-Level consumer doesn't need to be aware of higher level abstractions, such as `NakadiCursor`, which is where the timelines become relevant.

**Comment by adyach** [comment]
👍

File: api-consumption/src/main/java/org/zalando/nakadi/controller/EventStreamController.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
 import org.zalando.nakadi.exceptions.runtime.ServiceTemporarilyUnavailableException;
 import org.zalando.nakadi.exceptions.runtime.UnparseableCursorException;
 import org.zalando.nakadi.metrics.MetricUtils;
-import org.zalando.nakadi.repository.EventConsumer;
+import org.zalando.nakadi.repository.HighLevelConsumer;
```
[the remark this answers is not in the document]
Comment (adyach) [reply api-consumption/src/main/java/org/zalando/nakadi/controller/EventStreamController.java:38]: I moved highlevelconsumer

File: api-consumption/src/test/java/org/zalando/nakadi/controller/EventStreamControllerTest.java [hunk 2]
```diff
@@ -316,7 +317,7 @@ public void whenNoCursorsThenLatestOffsetsAreUsed() throws IOException, InvalidC
 
     @Test
     public void whenNormalCaseThenParametersArePassedToConfigAndStreamStarted() throws Exception {
-        final EventConsumer eventConsumerMock = mock(EventConsumer.class);
+        final HighLevelConsumer eventConsumerMock = mock(HighLevelConsumer.class);
```
[the remark this answers is not in the document]
Comment (adyach) [reply api-consumption/src/test/java/org/zalando/nakadi/controller/EventStreamControllerTest.java:320]: double checked, all good

**Comment by adyach** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**